### ISSUE TITLE
Upserted 0 pokemon, 0 pokestops, and 0 gyms

### ISSUE THREAD

**Issue by Plop** [AHAAAAAAA/PokemonGo-Map#2244]
Running python /var/www/PokemonGo/runserver.py -a google -u USERNAME -p PASSWORD -st 7 -P 5080 -L fr -t 1 -k APIKEY -l LOCATION, and I only got Upserted 0 pokemon, 0 pokestops, and 0 gyms

Nothing appears on the map.
![image](https://cloud.githubusercontent.com/assets/876470/17152984/ada3e14e-537a-11e6-8c1d-e7645bd1cce9.png)

Last commit.

Thanks for helping.

**Comment by Devastatin** [comment]
![english-working](https://cloud.githubusercontent.com/assets/20668837/17153322/263c11f2-534a-11e6-8b13-8c8a2046eac4.PNG)
Looks like the English version is working from my pull last night. first look at your login and make sure its properly working (the black dos box that shows the output). Then I'd try removing the Local and other options until you figure out.

**Comment by Plop** [comment]
Now I'm only running it with  python runserver.py  -P 5015 -l "Maurepas, France", and I still have this problem.

When changing location to a new one on my maps (tested with 15 differents maps), nothing appears.

**Comment by invisiblek** [comment]
weird, works here (that same location too)

try nuking your database and starting clean?

**Comment by michaelarlt** [comment]
@invisiblek unfurtunately it doesnt solve the issue

**Comment by C-5** [comment]
What the Number 2219  closed? 
Never  makes open. ..

**Comment by michaelarlt** [comment]
@C-5 what exactly are you referring to? the issue 2219? this would be related to the arguments which are content of the config. but @Plop isnt using the config at all since argument -se is not used.

**Comment by michaelarlt** [comment]
could be even similar to this one https://github.com/AHAAAAAAA/PokemonGo-Map/issues/2224

**Comment by Plop** [comment]
Well, it's now using the config in the config.ini file, and the argument override and the config.
So it should not be a problem.

Can't figure out what the problem is.

**Comment by michaelarlt** [comment]
@Plop i think its fine without using the config since youre using arguments. Even if youre using the config it might override the arguments given because the utils forces that as far as i know.
so running the instances just with arguments is totally fine. Also youre not the only one having this issue. I have several instances running and just two of them are having the behavior youre describing but atleast they upserting lured pokemons for whatever reasons. So i have lured pokemons but not wild pokemons for two areas out of ten .. wierd

**Comment by Plop** [comment]
The weirdest thing is that if I start the server with -l "Lille, France", it will not upsert anything.
If I then set the location with the WebGUI to "Acigné, France", everything will be upsert.

If I do the contrary (-l "Acigné, France"), it works fine for Acigné, but not for Lille if I set the location with the WebGUI.

Deleted the whole MySQL db, same problem.

**Comment by michaelarlt** [comment]
WTH .. i have tried that too now. It must be related to the PoGo Servers somehow. In several areas it doesnt upserting anything but in general it is upserting.

**Comment by Plop** [comment]
It would need more people to test that before we could tell what it is...

**Comment by michaelarlt** [comment]
just got the confirmation from some friend of mine. he cannot see anything on the map where i have everything but vice versa he does see pokemons and anything else in the area where i do not have any upserts. Now it becomes strange.
So can please more poeple test that?

**Comment by michaelarlt** [comment]
right now for me its wotking again. @Plop can you confirm?

**Comment by Plop** [comment]
Nope, absolutely the same problem for me @michaelarlt . Nothing changed at all

**Comment by Plop** [comment]
Lille has finally appeared. But the issue is still going on with EVERY other city in the entire world.

**Comment by michaelarlt** [comment]
ok .. i will test tomorrow again in different areas. Looks like, something is still strange.

**Comment by Plop** [comment]
@michaelarlt can you try with a freshly created Google account please ?
I did, and it seems to works fine now.

**Comment by Akabane87** [comment]
I have the same issue after deploying on jelastic... Nithing displayed for any locations (I tried multiple browsers on PC and phone so it's definitly a server issue). I redeployed 3 times mixing a new google account and default account proposed when deploying, and default API key/own API key. 

The weirdest thing is that the android port of this project is working fine with this same google account...

**Comment by Akabane87** [comment]
Ok I think I unedrstand the problem now. For some reason the settings entered when deploying on jelastic is not applyed to the command line in run config, so you have to edit the command line and restart the container in jelastic admin panel.

**Comment by Plop** [comment]
Still got the issue.

1)
Running python runserver.py -u FIRSTgmailaccount@gmail.com -P PORT -L "Strasbourg, France" -t 1 >>>>> upserted 0 pokemon, 0 gym, 0 stop

2)
Running python runserver.py -u SECONDgmailaccount@gmail.com -P PORT -L "Strasbourg, France" -t 1 >>>>> Got ALL informations

3)
Running python runserver.py -u FIRSTgmailaccount@gmail.com -P PORT -L "Strasbourg, France" -t 1 >>>>> upserted 0 pokemon, 0 gym, 0 stop

What could possibly be the issue ?!

**Comment by invisiblek** [comment]
@Plop banned?

**Comment by Plop** [comment]
@invisiblek I thought about it, even though some people told me it's impossible to be banned using the scanner. I then tried connecting the first google account on the game (for the very first time, I had to create my player, set a name, etc.

And I have interacted with a pokestop, could drop items from it. So... Not banned ?
But still not working on the map... I'm really lost there.

**Comment by invisiblek** [comment]
https://github.com/AHAAAAAAA/PokemonGo-Map/blob/master/README.md#warnings

**Comment by Plop** [comment]
Yeah, I know that, but if I can use Pokestops, it means I'm not banned, right ?

And some people told me even if I was banned, I could still scan the map.

**Comment by Plop** [comment]
Still having the issue, but I have more "informations" : 

Using the first account which is working at location A is OK. If I set it at location B or C, it's not working anymore.

Using the second account which is working at location A and B is fine. But then, changing location to location C, it's not working anymore.

A few hours later, the first account is working with location C. But not with A or B anymore.
Same for the second.

Where could possibly is the issue ?!
